content/en/cloud_cost_management/container_cost_allocation.md

```diff
@@ -16,56 +17,108 @@ further_reading:
-3. For AWS ECS support, set up [**Datadog Container Monitoring**][3] in ECS tasks.
-     - Optionally, enable [AWS Split Cost Allocation][5] for usage-based ECS allocation.
+{{< tabs >}}
+{{% tab "AWS" %}}
 
-## Cost allocation
+CCM allocates costs of all AWS EC2 instances across your container orchestration environments, including Kubernetes clusters and AWS-specific ECS clusters.
 
-Cost allocation divides host compute and AWS EBS volume costs from the cloud provider into associated individual tasks or pods. These divided costs are then enriched with tags from nodes, pods, tasks, and volumes (AWS only). This lets you break down costs by any associated dimensions.
+The following table presents the list of collected features and the minimal Agent and Cluster Agent versions for each.
 
-### Kubernetes
+| Feature | Minimal Agent version | Minimal Cluster Agent version |
+|---|---|---|
+| Container Cost Allocation | 7.27.0 | 1.11.0 |
+| AWS Persistent Volume Allocation | 7.46.0 | 1.11.0  |
 
-#### Compute
+1. Configure the AWS Cloud Cost Management integration on the [Cloud Costs Setup page][101].
+1. For Kubernetes support, install the [**Datadog Agent**][102] in a Kubernetes environment and ensure that you enable the [**Orchestrator Explorer**][103] in your Agent configuration.
+1. For AWS ECS support, set up [**Datadog Container Monitoring**][104] in ECS tasks. 
+1. Optionally, enable [AWS Split Cost Allocation][105] for usage-based ECS allocation.
 
-For Kubernetes compute allocation, a Kubernetes node is joined with its associated host instance costs. The node's cluster name and all node tags are added to the entire compute cost for the node. This allows you to associate cluster-level dimensions with the cost of the instance, without considering the pods scheduled to the node.
+[101]: https://app.datadoghq.com/cost/setup
+[102]: /containers/kubernetes/installation/?tab=operator
+[103]: /infrastructure/containers/orchestrator_explorer?tab=datadogoperator
+[104]: /containers/amazon_ecs/
+[105]: https://docs.aws.amazon.com/cur/latest/userguide/enabling-split-cost-allocation-data.html
 
-Next, Datadog looks at all of the pods running on that node for the day. The cost of the node is allocated to the pod based on the resources it has used and the length of time it ran. This calculated cost is enriched with all of the pod's tags.
+{{% /tab %}}
+{{% tab "Azure" %}}
 
-**Note**: Only _tags_ from pods and nodes are added to cost metrics. To include labels, enable labels as tags for [nodes][7] and [pods][8].
+CCM allocates costs of all Kubernetes clusters, including those managed through Azure Kubernetes Service (AKS). 
 
-#### Persistent volume storage (AWS only)
+1. Configure the Azure Cost Management integration on the [Cloud Costs Setup page][101].
+1. Install the [**Datadog Agent**][102] in a Kubernetes environment and ensure that you enable the [**Orchestrator Explorer**][103] in your Agent configuration.
```

Comment:
Very small nit, for AWS and GCP this comes after the version table below but in Azure it comes before